integration-tests/sjms-artemis-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+quarkus.artemis.enabled=true
```

Comment:
Isn't quarkus.artemis.enabled=true the default actually?

```suggestion

# Let Quarkus Artemis extension produce the ConnectionFactory
quarkus.artemis.enabled=true
```

Reply:
It have to enable explicitly see https://quarkiverse.github.io/quarkiverse-docs/quarkus-artemis/dev/index.html#_configuration_detection_at_build_time. It looks like changes for named configurations.